ops/terraform/env/prod-sbx/stateful/main.tf

```diff
@@ -29,4 +29,11 @@ module "stateful" {
   }
 
   victor_ops_url      = var.victor_ops_url
+
+  medicare_opt_out_config = {
+    # TODO: add read roles for DPC
+    read_roles        = []
+    write_roles       = ["arn:aws:iam::755619740999:role/bcda-dev-nfs-instance", "arn:aws:iam::755619740999:role/bcda-test-nfs-instance"]
+    admin_users       = ["arn:aws:iam::577373831711:user/DS7H", "arn:aws:iam::577373831711:user/VZG9", "arn:aws:iam::577373831711:user/BYSK"]
```

Comment:
Should we create a group for these users?

Reply:
I think that would be a good idea, but a good follow up question is where should this group be managed?